runtime/onert/core/include/ir/train/TrainingInfo.h

```diff
@@ -43,7 +43,7 @@ class TrainingInfo final
   const LossInfo &lossInfo() const { return _loss_info; }
   const OptimizerInfo &optimizerInfo() const { return _optimizer_info; }
   uint32_t batchSize() const { return _batch_size; }
-  uint32_t epoch() const { return _epoch; }
+  uint32_t &epoch() { return _epoch; }
```

Comment:
Could you add a member function that returns it by const reference? 
```suggestion
  const uint32_t &epoch() const { return _epoch; }
```